bundles/org.eclipse.vorto.core/src/org/eclipse/vorto/core/api/repository/ModelContent.java

```diff
@@ -23,30 +23,52 @@
  *	This class represents a resource that we can get from
  *  the repository. 
  */
-public interface IModelContent {
+public class ModelContent {
+	
+	private byte[] content;
```

Comment:
Model Content should not expose the content, because then the using component would need to know in what format it is. I would recommend to make the ModelContent and thus IModelRepository just be aware of Model objects and make it technology independent (Inpustream)